backend/index.js

```diff
@@ -14,7 +14,12 @@ if (process.env.NODE_ENV === 'development') {
 
 const server = http.createServer(app);
 const kubeconfig = initializeKubeconfig();
-const k8sUrl = new URL(kubeconfig.getCurrentCluster().server);
+
+const k8sUrl = new URL(
```

Comment:
I think it's better because it suggests the developer put this URL intentionally and the app will handle such situation.

`https://undefined:undefined` suggests that something went wrong and some variables are not loaded. Will the app work? Who knows...